packages/core/src/awsService/appBuilder/serverlessLand/metadataManager.ts

```diff
@@ -0,0 +1,122 @@
+/*!
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+import * as nodefs from 'fs' // eslint-disable-line no-restricted-imports
```

Comment:
Do we need to use the node specific one or can we use the internal `fs` module from `core/src/shared/fs/fs`

Reply:
I did try using `readFileText` method from Toolkit's  internal`fs` module, but it did not load the metadata properly.